### ISSUE TITLE
Loopback boot breaks with lodash@4.x.x

### ISSUE THREAD

**Issue by priyavivek1** [strongloop/loopback-boot#208]
Loopback boot breaks with following exception while using lodash 4.x.x and hence doesnt allow us to use lodash 4.

```
error: uncaughtException: Object has no method 'unique' date=Thu Sep 08 2016 12:25:20 GMT-0400 (Eastern Daylight Time), pid=8920, uid=null, gid=null, cwd=C:\projects\25-july\test, execP
ath=C:\Program Files\nodejs\node.exe, version=v0.10.39, argv=[node.exe, C:\projects\25-july\test], rss=86368256, heapTotal=71741888, heapUsed=41332856, loadavg=[0, 0, 0], uptime=13859.47
60019, trace=[column=19, file=C:\projects\25-july\test\node_modules\loopback-boot\lib\compiler.js, function=buildAllMixinInstructions, line=675, method=null, native=false, column=27, fil
e=C:\projects\25-july\test\node_modules\loopback-boot\lib\compiler.js, function=compile, line=89, method=null, native=false, column=22, file=C:\projects\25-july\test\node_modules\loopback-boot\index.js, function=bootLoopBackApp, line=135, method=null, native=false, column=1, file=C:\projects\25-july\test\server\server.js, function=
, line=207, method=null, native=false, column=26, file=module.js, function=Module.compile, line=456, method=_compile, native=false, column=10, file=module.js, function=Object.Module._extensions..js, line=474, method=Module.
extensions..js, native=false, column=32, file=module.js, function=Module.load, line=356, method=load, native=false, column=12, file=module.js, function=Function.Module._load, line=312, method=Module._load, native=false, colum
n=10, file=module.js, function=Function.Module.runMain, line=497, method=Module.runMain, native=false, column=16, file=node.js, function=startup, line=119, method=null, native=false, column=3, file=node.js, function=null, lin
e=935, method=null, native=false], stack=[TypeError: Object has no method 'unique', at buildAllMixinInstructions (C:\projects\25-july\test\node_modules\loopback-boot\lib\compiler.js
:675:19), at compile (C:\projects\25-july\test\node_modules\loopback-boot\lib\compiler.js:89:27), at bootLoopBackApp (C:\projects\25-july\test\node_modules\loopback-boot\index.js:135:22), at Object.<anonymous> (C:\projects\25-july\test\server\server.js:207:1), at Module._compile (module.js:456:26), at Object.Mod
ule._extensions..js (module.js:474:10), at Module.load (module.js:356:32), at Function.Module._load (module.js:312:12), at Function.Module.runMain (module.js:497:10), at startup (node.js:119:16), at node.j
s:935:3]
```

**Comment by superkhau** [comment]
Can you provide a sample for us to reproduce the issue? See https://github.com/strongloop/loopback/wiki/Reporting-issues#bug-report

**Comment by davidcheung** [comment]
possibly related https://github.com/strongloop/loopback-boot/pull/169

**Comment by bajtos** [comment]
This should have been fixed by #169. Please reopen this issue if the problem still persists.
